src/plugins/memdump/userhook.cpp

```diff
@@ -348,18 +348,28 @@ static bool make_trap(drakvuf_t drakvuf, drakvuf_trap_info* info, hook_target_en
     trap->name = target->target_name.c_str();
     trap->cb = target->callback;
     trap->data = target;
-    trap->breakpoint.lookup_type = LOOKUP_DTB;
-    trap->breakpoint.dtb = info->regs->cr3;
-    trap->breakpoint.addr_type = ADDR_VA;
-    trap->breakpoint.addr = exec_func;
+
+    page_info pageInfo;
+    int ret = vmi_pagetable_lookup_extended(vmi, info->regs->cr3, exec_func, &pageInfo);
```

Comment:
Why switch to doing this manualy? Keeping `LOOKUP_DTB` would have been fine.

Reply:
https://github.com/tklengyel/drakvuf/pull/687/files#diff-da50bc7c3b93e99cfbf61ebae9d8fc87R813
^ here we're looking for all traps that are present on the same physical page as CoW-ed address - that's why we want to store physical address rather than virtual.

Reply:
I see, that's a bit unconventional, usually info like that should be stored in the plugin itself, but I if you want to keep it the way it is at least add a comment saying so.